### ISSUE TITLE
Commands won't execute if Input Option to use "Cmd Enter" is selected

### ISSUE THREAD

**Issue by arougthopher** [samknight/slack_applescript#39]
In Slack, under Preferences->Advanced->Input Options ...

When writing a message, press <Enter> to ...

You can either choose "Send the Message", or "Start a new line (use Cmd Enter to send)"

This script expects the first option to be selected, and won't work if the second is.

**Comment by loupetron** [comment]
I tried changing the .scriptd to include `using {command down}` on each instance of `key code 36` but that didn't change anything for me. Haven't spent enough time with Apple Script to really know for sure.

I was able to get this to work using a new script, but I haven't been able to get the end time to work yet.

```
tell application "Slack"
	activate
	tell application "System Events"
		delay 0.5
		keystroke "/"
		delay 0.5
		keystroke "status :pika-dance: Back Tuesday"
		delay 0.5
		key code 36 using {command down}
		delay 0.5
		key code 36 using {command down}
	end tell
end tell
```

**Comment by samknight** [comment]
Thanks for raising the issue and the suggested fix. I have resolved this in the 2.0.3 release. Please let me know if you see any other issues
